go/engine/passphrase_recover.go

```diff
@@ -65,7 +65,8 @@ func (e *PassphraseRecover) Run(mctx libkb.MetaContext) (err error) {
 	}
 
 	// If the reset pipeline is not enabled, we'll want this to act exactly the same way as before
-	if !mctx.G().Env.GetFeatureFlags().HasFeature(libkb.EnvironmentFeatureAutoresetPipeline) {
+	// Autoreset hardcoded on for now.
+	if !true {
```

Comment:
long term we should get rid of these branches and make it appear as if the feature flag never existed